gossip/blockproc/evmmodule/evm.go

```diff
@@ -121,10 +119,7 @@ func (p *OperaEVMProcessor) Execute(txs types.Transactions) types.Receipts {
 }
 
 func (p *OperaEVMProcessor) Finalize() (evmBlock *evmcore.EvmBlock, skippedTxs []uint32, receipts types.Receipts) {
-	evmBlock = p.evmBlockWith(
-		// Filter skipped transactions. Receipts are filtered already
-		inter.FilterSkippedTxs(p.incomingTxs, p.skippedTxs),
-	)
+	evmBlock = p.evmBlockWith(inter.FilterSkippedTxs(p.incomingTxs, p.skippedTxs))
```

Comment:
nit: why is this comment removed? Is it wrong?

Reply:
Removed by mistake. Thanks